tasks/requirements_release_tasks.txt

```diff
@@ -1 +1,2 @@
-atlassian-python-api==3.41.3
\ No newline at end of file
+atlassian-python-api==3.41.3
+reno==3.5.0
```

Comment:
❓ question: How do we decide the version lof reno we use? This version is already 3 years old. Would it make sense to bump it in another PR?

Reply:
Yes I'm sure it would make sense. I would certainly encourage us to benefit from a dependabot thing to stay up to date